example/main.go

```diff
@@ -25,6 +25,6 @@ func main() {
 	mainServiceConfig := config.Get("services.main")
 
 	// Only to pretty print
-	mainServiceConfigPretty, err := json.MarshalIndent(mainServiceConfig, "", "  ")
+	mainServiceConfigPretty, _ := json.MarshalIndent(mainServiceConfig, "", "  ")
```

Comment:
why are you ignoring this error ?

Reply:
It was ignored before either way. Also it is only an example